### DISCUSSION TITLE
Set placeholder color of DatePickerInput

### DISCUSSION THREAD

**Discussion by telmen** [mantinedev/mantine#3975] [Q&A] [answered]
Hey, tt seems like DatePickerInput placeholder wrapped in a <span/>, but I can't find a way to style the placeholder via Styles API? Thanks!

**Comment by yverby** [comment, marked as answer]
You can use `placeholder` selector
```js
<DatePickerInput styles={{ placeholder: { ... } }} />
```

**Reply by telmen** [reply]
color property was being overriden by `'gray.5'`. had to add !important.

**Reply by yverby** [reply]
You are right, changing the color is not so easy, I looked at the sources, `DatePickerInput` component uses `InputPlaceholder` [component](https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Input/InputPlaceholder/InputPlaceholder.tsx) inside itself, and `InputPlaceholder` component uses `sx` prop directly, it has a higher priority, so the color cannot be overridden

You can use a workaround like [this](https://codesandbox.io/s/gifted-mendel-98xfd5?file=/src/App.tsx) or add to the color property `!important`

**Reply by telmen** [reply]
that works!
